src/Bundler.js

```diff
@@ -377,6 +379,9 @@ class Bundler extends EventEmitter {
     if (!this.watcher) {
       return;
     }
+    if (fs.existsSync(path) && fs.lstatSync(path).isSymbolicLink()) {
```

Comment:
I think we can turn watch into an async function and do this asynchronous?

Reply:
This is probably a good idea. I will take a look at it.

Reply:
I do not think a race condition could occur here, as the stats and exists function usually only take a few ms to execute, while the bundling takes one to two magnitudes longer.